js/main.js

```diff
@@ -80,26 +83,22 @@ function handleContent(event) {
 }
 
 function toggleNoEntries() {
-  const $h1 = document.createElement('h1');
+  console.log('length:', data.entries.length);
   if (data.entries.length === 0) {
-    $h1.innerText = 'THERE ARE NO ENTRIES RECORDED';
-    $ul.appendChild($h1);
+    $noEntry.classList.remove('hidden');
   } else {
-    const existing = $ul.querySelector('h1');
-    if (existing) {
-      existing.remove();
-    }
+    $noEntry.setAttribute('class', 'hidden');
```

Comment:
Change to using a global variable and to only check for the data.entries.length and show/hide the message